app/js/components/orderEntry/OrderEntryPage.jsx

```diff
@@ -300,34 +315,25 @@ export class OrderEntryPage extends PureComponent {
               note={this.props.note}
               location={this.props.location}
             />
-            <div className="header-nav">
-              <div>
-                <h3
-                  className="orders-nav"
-                  onClick={() => this.switchOrderType()}
-                  role="button">
-                  <b>
-                    <FormattedMessage
-                      id="app.orders.list"
-                      defaultMessage="Orders List"
-                      description="Orders List" />
-                  </b>
-                </h3>
-              </div>
-              {<SelectOrderType
-                switchOrderType={this.switchOrderType}
-                currentOrderType={this.props.currentOrderType}
-                page={page}
-              />}
-            </div>
-            <div className="body-wrapper drug-order-entry">
-              <RenderOrderType
-                backLink={returnUrl}
-                currentOrderTypeID={this.props.currentOrderType.id}
-                {...this.props}
-              />
-              {this.props.currentOrderType.id && this.renderDraftOrder()}
-            </div>
+            <Tabs
+              selectedIndex={this.state.index} onSelect={ (i) => {
+                this.switchTabs(i); this.setState({ index: i });
+              }
+              }>
+              <TabList>
+                <Tab><FormattedMessage id="app.orders.list" defaultMessage="Orders List" description="Orders List" /></Tab>
+                <Tab><FormattedMessage id="app.orders.labs.add" defaultMessage="Add Lab Orders" description="Add Lab Orders" /></Tab>
+              </TabList>
```

Comment:
@charlesdupuyrony I still think we are missing the Tab here for Drug Orders, see my comment here:

https://github.com/openmrs/openmrs-owa-orderentry/pull/230/files#r331563422

Let me know if I'm just missing it, and I can test further, thanks!

I believe you can see the SelectOrderType and RenderOrderType files for how this currently works.